gdcmodels/init_index.py

```diff
@@ -59,6 +59,25 @@ def format_index_name(prefix, index, index_type=None):
         return "_".join([prefix, index, index_type])
 
 
+def get_elasticsearch(args, use_ssl=True):
+    """Create an Elasticsearch client according to the given CLI args.
+
+    Args:
+        args: arguments. must have host, port, user, password attributes
+        use_ssl: Turn on/off SSL
+
+    Returns:
+
+    """
+    return Elasticsearch(
+        hosts=[{"host": args.host, "port": args.port}],
+        http_auth=(args.user, args.password),
+        use_ssl=use_ssl,
+        verify_certs=False,
+        timeout=60,
+    )
```

Comment:
FYI we should merge the changes from the Wocket branch so that we can more easily set the proper TLS settings. We should not be setting `verify_certs=False`. We should instead be providing a path to an appropriate CA bundle.